react_config.js

```diff
@@ -1,29 +0,0 @@
-const { overrides } = require('./.eslintrc.js')
-
-const ECMA_VERSION = 6
-
-module.exports = {
-  extends: './.eslintrc.js',
-  overrides: [
-    ...overrides,
-    {
-      files: ['*.jsx'],
-      parserOptions: {
-        ecmaVersion: ECMA_VERSION,
```

Comment:
I do not think `ecmaVersion: 6` was needed here. Please correct me if I am wrong about this.